website/cue/reference/components/sources/http_scrape.cue

```diff
@@ -0,0 +1,216 @@
+package metadata
+
+components: sources: http_scrape: {
+	title: "HTTP Scrape"
+	alias: "http_scrape"
+
+	classes: {
+		commonly_used: false
+		delivery:      "at_least_once"
+		deployment_roles: ["daemon", "sidecar", "aggregator"]
+		development:   "beta"
+		egress_method: "batch"
+		stateful:      false
+	}
+
+	features: {
+		acknowledgements: false
+		codecs: {
+			enabled:         true
+			default_framing: "`bytes`"
+		}
+		collect: {
+			checkpoint: enabled: false
+			from: {
+				service: services.http_scrape
+
+				interface: socket: {
+					direction: "outgoing"
+					protocols: ["http"]
+					ssl: "optional"
+				}
+			}
+			proxy: enabled: true
+			tls: {
+				enabled:                true
+				can_verify_certificate: true
+				can_verify_hostname:    true
+				enabled_default:        false
+			}
+		}
+		multiline: enabled: false
+	}
+
+	support: {
+		requirements: []
+		warnings: []
+		notices: []
+	}
+
+	installation: {
+		platform_name: null
+	}
+
+	configuration: {
+		auth: configuration._http_auth & {_args: {
+			password_example: "${PASSWORD}"
+			username_example: "${USERNAME}"
+		}}
+		endpoint: {
+			description: "Endpoint to scrape observability data from."
+			required:    true
+			warnings: ["You must explicitly add the path to your endpoint."]
```

Comment:
👍 relevant to my comment about the `configurable` description - we should try and make them like-for-like until we can deprecate the existing cue files.